copilot/fsd-account-store/manifest.yml

```diff
@@ -0,0 +1,55 @@
+# The manifest for the "account-store" service.
+# Read the full specification for the "Backend Service" type at:
+#  https://aws.github.io/copilot-cli/docs/manifest/backend-service/
+
+# Your service name will be used in naming your resources like log groups, ECS services, etc.
+name: fsd-account-store
+type: Backend Service
+
+# Distribute traffic to your service.
+http:
+  # Requests to this path will be forwarded to your service.
+  # To match all requests you can use the "/" path.
+  path: '/'
+  # You can specify a custom health check path. The default is "/".
+  healthcheck: '/healthcheck'
+
+# Configuration for your containers and service.
+image:
+  # Docker build arguments. For additional overrides: https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#image-build
+  location: ghcr.io/communitiesuk/funding-service-design-account-store:latest
+  # Port exposed through your container to route traffic to it.
+  port: 8080
+
+# Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html
+# Number of CPU units for the task.
+cpu: 256
+# Amount of memory in MiB used by the task.
+memory: 512
+
+# See https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#platform
+platform: linux/x86_64
+
+# Number of tasks that should be running in your service.
+count: 1
```

Comment:
Just a note to ask - how does this manifest change per environment?  (Really a question for later, but this count won't be 1 in the higher envs)

Reply:
Manifests are replaced via environment manifests. In copilot/environments/production/manifest.yml, we'd put in the overrides for this parameter there